spec/features/test_page_spec.rb

```diff
@@ -1,6 +1,116 @@
-feature 'test page' do
-  scenario 'visit test page' do
-    visit '/test'
-    expect(page).to have_content('test page')
+require_relative 'web_helpers'
+
```

Comment:
is there a spec for the Player class and the Game class?

Reply:
Not yet, this is something I was moving on to but should have been done first :)